_posts/en/newsletters/2021-05-19-newsletter.md

```diff
@@ -0,0 +1,162 @@
+  (CPFP)][topic cpfp] transaction from the transaction list screen, user-defined
+  fee amounts during coin selection, and various other improvements.
+
+- **Electrum 4.1.0 enhances Lightning features:**
+  [Electrum 4.1.0][electrum 4.1.0] adds [trampoline payments][topic trampoline payments], [multipath
+  payments][topic multipath payments], channel
+  backups, and other Lightning features. Additionally, this version of Electrum
+  supports [bech32m][topic bech32].
+
+- **BlueWallet v6.1.0 released:**
+  BlueWallet's [v6.1.0 release][bluewallet v6.1.0] adds Tor support,
+  [SLIP39][news86 slip39] support, and functionality for using [PSBTs][topic psbt]
+  with HD watch-only wallets.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.13.0-beta.rc2][LND 0.13.0-beta] is a release candidate that
+  adds support for using a pruned Bitcoin full node, allows receiving
+  and sending payments using Atomic MultiPath ([AMP][topic multipath payments]),
+  increased [PSBT][topic psbt] capabilities, among other improvements
+  and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+
+- [Bitcoin Core #21462][] adds tooling for attesting to outputs of Guix builds
+  and verifying these attestations against those of others. After this change,
+  Windows and macOS code signing remain the only missing piece before Guix
+  builds reach feature-parity with Gitian builds.
+
+- [Bitcoin Core GUI #280][] prevents displaying invalid Bitcoin
+  addresses in an error dialog, eliminating the ability to display an
+  arbitrary message in an official-looking dialog.  A simple "invalid
+  address" error is now displayed instead.  (See the PR for illustrative
+  before and after screenshots.)
+
+- [Bitcoin Core #21359][] updates the `fundrawtransaction`, `send` and
+  `walletcreatefundedpsbt` RPCs with a new `include_unsafe` parameter
+  that can be used to spend unconfirmed UTXOs created by other users in
+  the transaction.  This allows fee bumping a transaction using
+  [CPFP][topic cpfp] and was added for that reason by a developer
+  working on implementing [anchor outputs][topic anchor outputs] in the
+  Eclair LN node.  The option should only be used when necessary, as
+  unconfirmed transactions created by other users can be replaced, which
+  may prevent any child transactions from being confirmed.
+
+- [LND #5291][] improves the way LND ensures that [PSBTs][topic psbt] for
```

Comment:
"LND PR improves the way LND does thing ..." seems just a little awkward. Perhaps this sentence can be reformulated to remove the repetitious LND.